tracer/test/test-applications/debugger/dependency-libs/Directory.Build.props

```diff
@@ -1,6 +1,18 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
+  <PropertyGroup>
+    <!-- Hide warnings for EOL .NET Core targets (e.g. netcoreapp3.0) -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <!-- Stop NuGet complaining about us targetting obsolete TFMs etc-->
+    <NuGetAudit>false</NuGetAudit>
+    <!--  _Really_ stop it complaining  -->
+    <NoWarn>NU1901;NU1902;NU1903;NU1904</NoWarn>
+
+    <ArtifactsPath>$(MSBuildThisFileDirectory)../../../../../artifacts</ArtifactsPath>
+    <UseArtifactsOutput>true</UseArtifactsOutput>
+  </PropertyGroup>
+
```

Comment:
Isn't this already imported (with `<Import>` above) from the file one directory above this one?

Reply:
Yeah, you're right. We still need to update the artifacts path, but we can ignore the rest 👍